### PR TITLE
rhino website 3.0

### PR DESCRIPTION
**Pull request by oklopfer**

### CODE REVIEW

File: src/components/index/quote.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        quote: "Rhino Linux's latest release makes it the Swiss Army knife of Linux distributions.",
+        author: "Jack Wallen",
+        link: "https://www.zdnet.com/article/rhino-linux-latest-release-makes-it-the-swiss-army-knife-of-linux-distributions/",
+        source: "ZDNET"
+    },
+];
+
+function Quote() {
+    return (
+        <div className="md:w-[85%] m-auto grid grid-cols-1 gap-4 pb-8 md:pt-4">
```
Comment (ajstrongdev) [inline src/components/index/quote.jsx:20]: Please change to use grid-cols-2 on medium+large devices pls.

File: src/components/main/footer.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+const footerItemsB = [
+    { name: 'Code of Conduct', href: '/code/'},
+    { name: 'Donate', href: 'https://ko-fi.com/rhinolinux' },
+];
+
+function Footer() {
+    return(
+        <footer className="p-8">
+            <hr className="md:w-[85%] h-1 mx-auto my-4 bg-site-300 border-0 rounded" />
+            <div className="md:w-[85%] m-auto flex items-center justify-center">
```
Comment (ajstrongdev) [inline src/components/main/footer.jsx]: All footer items should stack and be center-aligned on mobile

File: src/components/download/selection.jsx [hunk 3]
```diff
@@ -18,43 +18,60 @@ const DownloadSelection = () => {
...
-      </select>
+    <div className="md:w-[85%] w-full mx-auto md:p-0 p-4">
+      <h1 className="text-off-white text-center pt-8 text-3xl drop-shadow-header-drop">
+        Download Rhino Linux
+      </h1>
+      <h2 className="text-off-white text-center text-2xl font-light mt-2 drop-shadow-header-drop">
+        Version <b><b>2024.2</b></b>
+      </h2>
+      <div className="mx-2">
+        <select
```
Comment (ajstrongdev) [inline src/components/download/selection.jsx:57]: Arrow overlaps with text on mobile view

**Review by ajstrongdev — CHANGES_REQUESTED** [review]

File: src/components/download/selection.jsx [hunk 4]
```diff
@@ -18,43 +18,60 @@ const DownloadSelection = () => {
...
+              <span className="tooltiptext" id={content.title}>Copy to clipboard</span>
+              <button
+                onClick={() => {
+                  navigator.clipboard.writeText(content.details.shasum);
+                  document.getElementById(content.title).innerHTML = "Copied to clipboard!";
+                }}
+                onMouseOut={() => {
+                  document.getElementById(content.title).innerHTML = "Copy to clipboard";
+                }}
+             className="my-4 hover:scale-105 transition-all bg-site-100 p-1 rounded-[0.65em] text-white text-sm px-2 py-1 shadow-md">sha256sum</button>
```
Comment (ajstrongdev) [inline src/components/download/selection.jsx:42]: Fix padding on button to make it match the download button.

Comment (oklopfer) [reply src/components/download/selection.jsx:42]: Padding does match proportionately

File: src/components/main/footer.jsx (hunk 2 above)
Comment (oklopfer) [reply src/components/main/footer.jsx]: That’s way too much space for the footer to take up

File: src/components/download/selection.jsx (hunk 3 above)
Comment (oklopfer) [reply src/components/download/selection.jsx:57]: does on current rhino site too I’ll try to make the text resize

File: src/components/index/quote.jsx (hunk 1 above)
Comment (oklopfer) [reply src/components/index/quote.jsx:20]: Sure

File: public/img/blobs.png
Comment (ajstrongdev) [inline public/img/blobs.png:1]: Convert to webp.

File: README.md [hunk 5]
```diff
@@ -5,13 +5,7 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
...
 
 ```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
+npx concurrently "pnpm run dev" "npx tailwindcss -i ./src/styles/src/input.css -o ./src/styles/output.css --watch"
```
Comment (ajstrongdev) [inline README.md:8]: not everyone uses pnpm, change to npm.

File: public/img/ecosystem/applications/rhino-pkg.png
Comment (ajstrongdev) [inline public/img/ecosystem/applications/rhino-pkg.png:1]: Convert to webp.

File: public/img/ecosystem/applications/setup-wizard.png
Comment (ajstrongdev) [inline public/img/ecosystem/applications/setup-wizard.png:1]: Convert to webp.

File: public/img/ecosystem/applications/your-os.png
Comment (ajstrongdev) [inline public/img/ecosystem/applications/your-os.png:1]: Convert to webp

File: public/img/home/developers.png
Comment (ajstrongdev) [inline public/img/home/developers.png:1]: Convert to webp.

File: public/img/home/ulauncher-a.png
Comment (ajstrongdev) [inline public/img/home/ulauncher-a.png:1]: Convert to webp.

File: public/img/home/ulauncher-b.png
Comment (ajstrongdev) [inline public/img/home/ulauncher-b.png:1]: Convert to webp.

File: public/img/orb.png
Comment (ajstrongdev) [inline public/img/orb.png:1]: Convert to webp.

File: public/img/unicorn/appgrid.png
Comment (ajstrongdev) [inline public/img/unicorn/appgrid.png:1]: Convert to webp.

File: public/img/unicorn/tiling.png
Comment (ajstrongdev) [inline public/img/unicorn/tiling.png:1]: Convert to webp.

File: public/img/unicorn/xfdashboard.png
Comment (ajstrongdev) [inline public/img/unicorn/xfdashboard.png:1]: Convert to webp

File: src/components/download/selection.jsx [hunk 6]
```diff
@@ -18,43 +18,60 @@ const DownloadSelection = () => {
...
+          className="p-2 px-4 bg-site-300 w-[100%] text-off-white select-fix md:text-2xl font-light rounded-[0.5em] mt-4 appearance-none"
+        >
+          <option value="generic">Generic (x86_64/ARM64)</option>
+          <option value="pine">Pine64 (ARM64)</option>
+          <option value="rpi">Raspberry Pi (ARM64)</option>
+        </select>
+      </div>
 
-      <div className="content-display mt-6">
+      <div className="content-display mt-6 flex-grid">
```
Comment (ajstrongdev) [inline src/components/download/selection.jsx]: remove mt-6, it adds too much vertical padding between the selection box and the downloads. 
Padding issues

File: src/components/unicorn/content.jsx [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+import Image from 'next/image';
+import Link from 'next/link';
+
+// Array holding the data for each section
+const contentData = [
+  {
+    title: "Adaptive, to any workflow",
+    description:
+      " Whether you prefer the traditional floating windows, or are a fanatic of tiling window managers, Unicorn is perfect for you. With optional auto-tiling you can easily toggle between floating and tiling at the press of a button.",
+    imgSrc: "/img/unicorn/tiling.png",
```
Comment (ajstrongdev) [inline src/components/unicorn/content.jsx:11]: webp.

File: src/components/unicorn/content.jsx [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+    buttonText: "Learn the keybinds",
+    buttonLink: "https://wiki.rhinolinux.org/user/unicorn",
+  },
+  {
+    title: "A beautiful app grid",
+    description:
+      "Our application grid, Lightpad, is a simple and powerful application launcher that is Wayland compatible. We have made modifications to the theming of Lightpad to integrate perfectly into Unicorn and Rhino Linux.",
+    imgSrc: "/img/unicorn/appgrid.png",
```
Comment (ajstrongdev) [inline src/components/unicorn/content.jsx:19]: webp

File: src/components/unicorn/content.jsx [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+    buttonText: null,
+    buttonLink: null,
+  },
+  {
+    title: "Modern desktop management",
+    description:
+      "You can switch between virtual desktops with ease. We have loaded in Xfdashboard, with many minor tweaks and improvements. Xfdashboard brings a gnome-shell like virtual desktop and exposé built right into XFCE.",
+    imgSrc: "/img/unicorn/xfdashboard.png",
```
Comment (ajstrongdev) [inline src/components/unicorn/content.jsx:27]: webp

File: src/pages/index.jsx [hunk 10]
```diff
@@ -1,21 +1,24 @@
...
   return (
     <main>
-      <Menu />
+      <SiteData title="Homepage" description="An Ubuntu-based, rolling release distribution, with Pacstall & XFCE at its core." />
       <Hero />
-      <Latest />
-      <Developers />
-      <Desktop />
-      <Packages />
+      <Announcement />
```
Comment (ajstrongdev) [inline src/pages/index.jsx]: 2024.2 has been out for a while, please comment this out until we need to use the announcement banner again.

File: src/styles/src/input.css
Comment (ajstrongdev) [inline src/styles/src/input.css:1]: Just a question: why did you change this from `globals.css` ? Was there a reason.

**Review by ajstrongdev — CHANGES_REQUESTED** [review]
Mostly minor changes, convert all `png` images to `webp` + fix small padding issues

File: src/styles/src/input.css
Comment (oklopfer) [reply src/styles/src/input.css:1]: Consistency

**Review by ajstrongdev — APPROVED** [review]
LGTM, ready to merge
